Dev-2/Dev-2/ChangeStringTransliteration.cs

```diff
@@ -0,0 +1,204 @@
+            {
+                str = str.ToLower();
+            } catch
+            {
+                throw new Exception("Can't convert <String> to lower case");
+            }
+            return true;
+        }
+        /// <summary>
+        /// check if string contain cyrillic letters
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns>
+        /// return true if string contain only cyrillic letters
+        /// return false if string conatain any other letters
+        /// </returns>
+        private bool IsCyrillic (String str)
+        {
+            foreach (char i in str)
+            {
+                if (char.IsLetter(i) && (!Enumerable.Range(1072, 1103).Contains(i)))
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+        /// <summary>
+        /// check if string contain latin letters
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns>
+        /// return true if string contain only Latin letters
+        /// return false if string conatain any other letters
+        /// </returns>
+        private bool IsLatin (String str)
+        {
+            foreach (char i in str)
+            {
+                if (char.IsLetter(i) && (!Enumerable.Range(97, 122).Contains(i)))
+                {
+                    return false;
+                }
+            }
+            return true;
+        }
+        /// <summary>
+        /// Autodect which alphabet string contain 
+        /// change transliteraton of string <param>
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns>return transliterated string</returns>
+        public String ChangeTransliteration(String str)
+        {
+            String temporaryString = String.Empty;
+            temporaryString = str;
+            try
+            {
+                StringToLowerCase(ref temporaryString);
+
+                if (IsCyrillic(temporaryString))
+                {
+                    CyrillicToTransliteration(ref temporaryString);
+                }
+                else if (IsLatin(temporaryString))
+                {
+                    LatinToTransliteration(ref temporaryString);
+                }
+                else
+                {
+                    Console.WriteLine("The initial sting conains russian and english symbols");
+                }
+                return temporaryString;
+            }
+            catch 
+            {
+                throw new Exception("Can't translit this tring");
+            }
+        }
+        /// <summary>
+        /// replace each element in string ,using the dictionary
+        /// </summary>
+        /// <param name="str">reference on string</param>
+        private  void CyrillicToTransliteration (ref String str)
+        {
+            StringBuilder temporaryString = new StringBuilder();
+            temporaryString.Append(str);
+
```

Comment:
Dictionary initialization has to be out of method not to reinitialize it each time you call method